CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to Celestia
+
+Welcome and Thank You for contributing to building Celestia, the first modular
+blockchain network. 
+
+In this guide you will find information about how Celestia manages the
+`CelestiaOrg` Github as well as expectation around our Engineering Standards.
+
+## Github Management
+
+The `CelestiaOrg` Github uses teams to manage access to the organization and
```

Comment:
Are we actually going to use groups? Last time we failed and abandoned this idea.

Reply:
yes, this is on my todo list